### ISSUE TITLE
Print Token Page Incomplete

### ISSUE THREAD

**Issue by aarontitus** [CrisisCleanup/crisiscleanup-4-web#117]
1. Go to https://app.staging.crisiscleanup.io/print_token/892f7eae-d301-4658-ad51-06a48430330d on a smart phone (FF, Android)
2. Bug: Page is wider than the screen.
3. Bug: There is no ability to close individual work types or change their statuses
4. Bug: Remove leading comma in Address.

Bugs: (i.e. left-right scroll, no work type information)
![Image](https://user-images.githubusercontent.com/4184619/223792740-cdbc0e58-3114-42f5-86a8-ebee2d639770.jpg)

Should look like:
![Image](https://user-images.githubusercontent.com/4184619/223792947-6969eadc-2d2b-48e2-a193-c2d70ebd38fc.png)

**Comment by aarontitus** [comment]
New Bugs:
1. I visited https://app.staging.crisiscleanup.io/print_token/892f7eae-d301-4658-ad51-06a48430330d in FF, in a private window.
2. The page initially loaded and I saw two work type dropdowns. Bug: However, the dropdowns contained no status labels, and the dropdowns themselves were empty.
3. Bug: After a few seconds, the system forwarded me to a login page. Unable to reload, even in new private windows.

I suspect that the page is asking for authentication now, and can't load status information without auth.
Changed to Blocker.
